### PR TITLE
Car details

### PR DESCRIPTION
**Pull request by Pratishah**
Added design  and test cases in CarDetails Page
added book now button at the bottom

Ticket Link : https://abhirams.atlassian.net/browse/P3ECWFC-48 
                    https://abhirams.atlassian.net/browse/P3ECWFC-49

Screenshot (21)
Screenshot (22)
Screenshot (23)
e

### CODE REVIEW

File: src/pages/CarDetails/CarDetails.scss [hunk 1]
```diff
@@ -1,32 +1,209 @@
...
 }
 .detailsContainer {
-  img {
-    display: block;
-    width: 100%;
-  }
+  // img {
+  //   display: block;
+  //   width: 100%;
+  // }
```
Comment (pashedin) [inline src/pages/CarDetails/CarDetails.scss]: remove unnecesaary comments, please

File: src/pages/CarDetails/CarDetails.scss [hunk 2]
```diff
@@ -1,32 +1,209 @@
+@import "./common-fontstyles.scss";
+@import "../../themes/colors.scss";
+
 .totaldiv {
-  margin: 2rem;
+  padding-top: 1rem;
+  border-left: 8vh solid black;
+  border-right: 8vh solid black;
```
Comment (pashedin) [inline src/pages/CarDetails/CarDetails.scss:8]: why do we need to use vh??

File: src/pages/CarDetails/CarDetails.tsx [hunk 3]
```diff
@@ -21,116 +24,277 @@ function CarDetails() {
...
-                width: "10rem",
-                backgroundColor: car?.exterior.color,
-                margin: "0.5rem",
-                border: "0.1rem solid black",
-              }}
-            />
-            <p>
-              <span>Dimension</span>
+    <>
+      <Container id="top"></Container>
```
Comment (pashedin) [inline src/pages/CarDetails/CarDetails.tsx]: why do we need to use the Container?

File: src/pages/CarDetails/CarDetails.tsx [hunk 4]
```diff
@@ -21,116 +24,277 @@ function CarDetails() {
...
+      <Container fluid className="totaldiv">
```
Comment (pashedin) [inline src/pages/CarDetails/CarDetails.tsx]: `totalDiv` Please change this to kebab-case

File: src/pages/CarDetails/CarDetails.tsx [hunk 5]
```diff
@@ -21,116 +24,277 @@ function CarDetails() {
...
+        <div className="black-box-left"></div>
+        <Container className="detailsContainer">
```
Comment (pashedin) [inline src/pages/CarDetails/CarDetails.tsx]: All class names in kebab case please

**Review by pashedin — CHANGES_REQUESTED** [review]
Please fix the following, before I continue with the review
- unnecessary comments
- inline styling
- class names in kebab-case

File: src/pages/CarDetails/CarDetails.scss (hunk 2 above)
Comment (Pratishah) [reply src/pages/CarDetails/CarDetails.scss:8]: to get the left and right vertical black border i have used 8vh 

image

File: src/pages/CarDetails/CarDetails.tsx [hunk 6]
```diff
@@ -21,116 +22,191 @@ function CarDetails() {
...
+                  <p>
+                    <span>Top Speed</span>
+                    <br />
+                    <p> {car?.specifications.top_speed}</p>
+                  </p>
+                  <p>
+                    <span>Variants</span>
+                    <br />
+                    {car?.specifications.variant.map((v) => (
+                      <li key={++key}>{v}, </li>
```
Comment (pashedin) [inline src/pages/CarDetails/CarDetails.tsx:90]: bad way to use key but okay for now

**Review by pashedin — CHANGES_REQUESTED** [review]
how are all 3 Bar.svg same???
Please use the correct component image, its okay if this image is static. I see you have used CSS ot modify it, good effort but ther was no need.

**Review by pashedin — APPROVED** [review]

**Pull request merged**